### PR TITLE
store BundleAndSplitMetadata in metastore

### PR DESCRIPTION
**Pull request by PSeitz**
- store BundleAndSplitMetadata in metastore
- rename
- fmt

### Context / purpose
Describe the context and purpose of this PR. Link related issue(s) if any.

### Description
Describe the proposed changes made in this PR.

### How was this PR tested?
Describe how you tested this PR.

### Checklist
*Remove or complete this list if required.*
- [x] tested locally
- [ ] added unit tests
- [x] included documentation

### CODE REVIEW

File: quickwit-metastore/src/metastore.rs [hunk 1]
```diff
@@ -49,6 +49,15 @@ pub struct IndexMetadata {
     pub checkpoint: Checkpoint,
 }
 
+/// Carries split and bundle offsets for single read metadata.
+#[derive(Clone, Eq, PartialEq, Default, Debug, Serialize, Deserialize)]
+pub struct BundleAndSplitMetadata {
```
Comment (evanxg852000) [inline quickwit-metastore/src/metastore.rs:54]: Can we probably later find a better name for `BundleAndSplitMetadata` ?

**Review by evanxg852000 — APPROVED** [review]
Looks :+1: , maybe we can find a better name for  `BundleAndSplitMetadata` later

**Comment by guilload** [comment]
Why we are duplicating `bundle_offsets` in `BundleAndSplitMetadata` and again in `SplitMetadata`?  I removed the former on a local branch, and the project compiles just fine. I must be missing something.

**Comment by PSeitz** [comment]
I forgot to remove the offsets from `SplitMetadata`. It's not yet used in search since we decided to split up the PR in multiple.

**Pull request merged**